tests/installation/performing_installation/perform_installation.pm

```diff
@@ -0,0 +1,21 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Wait for the installation to be finished.
+# Use TIMEOUT_SCALE so expected installation time can be adjusted
+# for slower architectures.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi 'get_var';
+
+sub run {
+    my $timeout = 2750 * get_var('TIMEOUT_SCALE', 1);
```

Comment:
I ve also seen elsewere this usage of integer numbers directly multiplying the TIMEOUT_SCALE but maybe it could be more explanatory if `2750` would be declared beforehand as a variable or constant, naming it something like `default_install_duration`?
If not, at least a comment along the lines of 2750 seconds is the typical installation duration in our setup.
It would be good to not have this constant as an obscure (for a non initiated reader) magic number.

Reply:
It is a timeout, it is not the expected time of duration, that would require further analysis (you know, like some expected time average plus some margin or something like that, with some record message when it reached the expected value for further investigation...) and as we discussed offline we are not so concern about performance to apply this changes, we should report it to the proper team if needed. So it is a magic number indeed, like any other timeout long enough to be safe, it is so obscure that in fact it should be 5500 seconds. I'm adding the comment (I had it there for a time and I deleted) but only for the reason that I wasn't sure if we would fix our client on time to represent the right value before this is merged. Anyway good point, we might not set there 5500 for a while and only we know why is that without a comment.

Reply:
https://progress.opensuse.org/issues/103548